app/models/filter.rb

```diff
@@ -102,7 +102,7 @@ def to_label
 
   def resource_type
     type = @resource_type || filterings.first.try(:permission).try(:resource_type)
-    type.presence
+    type || '(Miscellaneous)'
```

Comment:
In the filters when resource_type = null it shows "Miscellaneous", but this can't work in the breadcrumb switcher.
When the breadcrumb switcher component gets null in the name it will not show the item at all.

Reply:
keep in mind that this probably  breaks the tests
The `(Miscellaneous)` needs to be translated